public/app/features/browse-dashboards/components/BrowseActions/BrowseActions.tsx

```diff
@@ -144,3 +147,19 @@ const getStyles = (theme: GrafanaTheme2) => ({
     marginBottom: theme.spacing(2),
   }),
 });
+
+function trackAction(action: 'move' | 'delete', selectedDashboards: string[], selectedFolders: string[]) {
+  const interactionIdentifier =
+    action === 'move' ? 'grafana_manage_dashboards_item_moved' : 'grafana_manage_dashboards_item_deleted';
+  const itemTypes = [];
+  if (selectedFolders.length > 0) {
+    itemTypes.push('folder');
+  }
+  if (selectedDashboards.length > 0) {
+    itemTypes.push('dashboard');
+  }
+  reportInteraction(interactionIdentifier, {
+    item_type: itemTypes.join(','),
```

Comment:
@zsofiakm up2u, trivial to do either so whatever's easiest for you to query 👍

Reply:
If it's an array, does that mean that it will appear in BigQuery as separate lines? That would be great.

Reply:
Good call @joshhunt 

Actually, I believe if you send it an array it will be stored as a string anyways ... so not super fun to query it afterwards (having fun with SQL + REGEXP)

If we add count props instead (`{folders: 2, dashboards:0}`) it will be wayyyy easier 😊 - I've made that mistake myself and updated it after trying to make the dashboard 😅  (https://github.com/grafana/grafana/pull/52052)

Reply:
sounds good, i'll add counts 👍

Reply:
@yaelleC @zsofiakm just wanna be explicit about something as well... these counts will be different to the counts shown in the move/delete modals. idk what analytics you wanna extract from this data, but those counts are showing everything that's _affected_ by a move, whereas these counts will be the actual items moved in the backend.

so if you move a folder containing 5 subfolders, in the modal we would show 6 folders affected, but in the backend (and in this rudderstack event) we would show only 1 folder moved. 

lmk if that's ok/what's expected, else i'll have to do some refactoring

Reply:
I personally don't think I'll be using the counts, at least I can't think of a use case. So we can add what we have.